package.json

```diff
@@ -5,8 +5,8 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "lint": "belly-button -f",
-    "test": "npm run lint && lab -v -a code -p -t 100"
+    "lint": "belly-button -i lib/*.js -f",
```

Comment:
I'm not 100% sure, but I think `lib/*.js` needs to be put in single quotes. Otherwise, I think the shell will expand the `*` before running the command.